docker/Readme.md

```diff
@@ -43,3 +43,8 @@ When making changes in the local source tree, the plugin can be rebuilt/resintal
 ```bash
 ./rebuild.sh
 ```
+
+Build the binary rpm, e.g. for installation on production system:
+```bash
+docker compose exec -T slurm bash < make-rpm.sh
```

Comment:
Should this be: `< make-binary-rpm.sh`?

Reply:
I'm sorry, I forgot to push the last commit. It is fixed now.